packages/round-manager/src/features/api/payoutStrategy/merklePayoutStrategy.ts

```diff
@@ -0,0 +1,36 @@
+import { ethers, Signer } from "ethers";
+import { merklePayoutStrategy } from "../contracts";
+
+/**
+ * Deploys a QFVotingStrategy contract by invoking the
+ * create on QuadraticFundingVotingStrategyFactory contract
+ *
+ * @param signerOrProvider
+ * @returns
+ */
+export const deployMerklePayoutStrategyContract = async (
+  signerOrProvider: Signer
+): Promise<{ payoutContractAddress: string }> => {
+  try {
+    const payoutStrategyFactory = new ethers.ContractFactory(
+      merklePayoutStrategy.abi,
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      merklePayoutStrategy.bytecode!,
```

Comment:
I didn't know you could deploy a new contract like this, super cool! 😄

Reply:
yes yes :D this library is a saviour  
https://docs.ethers.io/v5/api/contract/contract-factory/#ContractFactory-deploy